### ISSUE TITLE
[Question] Two questions regarding the PPO implementation

### ISSUE THREAD

**Issue by pengzhi1998** [DLR-RM/stable-baselines3#917]
Hi, thank you again for this great implementation! However, I have two questions regarding PPO:

1. I found that you have provided a function named `CheckpointCallback` to save the trained model for every specific number of iterations. However, how may I save the model with the best performance (highest mean accumulative reward during rollout) with any provided functions? Is there an argument to achieve this?

2. Secondly, I found in my task, that it's important to improve PPO's exploration ability. I'm thinking of executing some more "aggressive" actions (improve the std value when sampling from the Normal distribution during training). But it seems that this is a trainable parameter in your implementation, and after a few iterations, the output std value for sampling an action from the non-deterministic policy will be rather small. In that case, the agent would have a smaller chance to pick up random actions. May I have your suggestions on this? Just change [requires_grad](https://github.com/DLR-RM/stable-baselines3/blob/master/stable_baselines3/common/distributions.py#L140) to be false?

Thank you!

**Comment by araffin** [comment]
Hello,
your two questions can be achieved with callbacks, you should take a look at the callback section in the doc and the associated tutorial. Your question 2 was already answered in https://github.com/DLR-RM/stable-baselines3/issues/231

**Comment by pengzhi1998** [comment]
Thank you so much! I'll take a look.
